bash_completion.sh

```diff
@@ -83,7 +83,7 @@ _buildtest ()
 
   COMPREPLY=()   # Array variable storing the possible completions.
 
-  local cmds="build buildspec cd cdash clean config debugreport docs edit help inspect history path report schema schemadocs stylecheck unittests"
+  local cmds="build buildspec cd cdash clean config debugreport docs edit-test help inspect history path report schema schemadocs stylecheck unittests"
```

Comment:
please remove `edit-test` from here these are the top level commands  such as `buildtest build`. The `buildtest buildspec edit-test` is a subcommand which is followed in separate case statement.